src/Microsoft.Compatibility.HashCode/ref/Configurations.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
```

Comment:
Can we use `netstandard2.0` instead of `netstandard` here?  I'd prefer we start to work towards the versioned targetGroups in packages.